### ISSUE TITLE
FreakyPicker ignores BackgroundColor

### ISSUE THREAD

**Issue by janusw** [FreakyAli/Maui.FreakyControls#158]
### Description

The `FreakyPicker` control seems to ignore the property `BackgroundColor` completely. There is no visual change if I add it.

I have only observed this on Android so far. Have not checked iOS.

### Code

To reproduce, simply add a `BackgroundColor` to the picker in the sample (at tag v0.5.0).

```
            <freakyControls:FreakyPicker
                Title="Select something :D"
                <!-- some more propeties ... -->
                TextColor="Black"
                TitleColor="LightGray"
                BackgroundColor="Purple"/>
```

### Expected Behavior

The background should be colored in the requested color.

### Actual Behavior

It does not change.

### Basic Information

- Version with issue:  0.5.0
- Last known good version:  ?
- IDE:  VSCode on Mac
- Platform Target Frameworks: 
  - Android: Android 14 / API level 34
  - iOS:  ?
  - Dotnet:  8.0.402
- Target Devices:
  -   Samsung Galaxy S21

**Comment by FreakyAli** [comment]
Can you also check if this bug exists on the Maui picker of control

**Comment by janusw** [comment]
Yes, on the standard MAUI `Picker` I can change the `BackgroundColor` without any problem.

**Comment by FreakyAli** [comment]
@janusw  I checked this, it seems that when you set the background to transparent (for the underline), it removes it altogether and does not let you change the color again, adding it back brings back the underline...

**Comment by janusw** [comment]
> [@janusw](https://github.com/janusw) I checked this, it seems that when you set the background to transparent (for the underline), it removes it altogether and does not let you change the color again

You mean this here?

https://github.com/FreakyAli/Maui.FreakyControls/blob/56e033ccc79385bc439065cd2e7d66ddcc2babb8/MAUI.FreakyControls/MAUI.FreakyControls/Platforms/Android/FreakyPickerHandler.android.cs#L18-L19

> adding it back brings back the underline...

So what you're saying is: You're setting the background to transparent on Android, in order to make the underline disappear? Therefore the inability to change the background color is a side-effect of your implementation on Android. Correct so far?

If yes, then I guess these are the options for dealing with it:
* Override the `BackgroundColor` property to make it read-only / not settable (if possible?).
* Or: At least document the fact that the `BackgroundColor` is not changeable for `FreakPicker` on Android.

**Comment by FreakyAli** [comment]
@janusw Yes, this seems to be why it's happening, 

>Override the BackgroundColor property to make it read-only / not settable (if possible?).

I was thinking of doing the same thing, maybe even re-implement how it works and see if that does anything for Android 

>Or: At least document that the BackgroundColor is not changeable for FreakPicker on Android.

It's not just the picker, more controls are using this like Entry, DatePicker, TimePicker and others....

**Comment by FreakyAli** [comment]
For the time being you can wrap it in a Layout and give it a background color if you want a Color other than transparent

**Comment by FreakyAli** [comment]
Closing this since it's a native thing that cannot be changed, pining it just in case!

**Comment by FreakyAli** [comment]
I already tried every possible way to fix the bug and the issue is the .NET MAUI api that sets the background color resets the Background property so as long as I can't change the internal implementation of the Background property this is unfixable, I'm sure they won't be changing that just to support my library 🙂

If you think this is so easy to fix you can give it a try but I'm sure you have better things to do, anyways I was going to document it ergo I pinned the issue, as I do this in my free time and it's an unpaid project, anyways I am sure you're correct that my library is unfit for use, thanks for your feedback really appreciate it 🙂
